app/views/trains/index.html.slim

```diff
@@ -1,5 +1,9 @@
 h1 Trains Index
 = link_to 'New train', new_train_path
+br
+- if notice
```

Comment:
can we put the `br` inside the if condition too?
just so we dont have a free floating `br` with no content